django_ipam/base/models.py

```diff
@@ -0,0 +1,19 @@
+import swapper
+from django.db import models
+
+
+class AbstractSubnet(models.Model):
+    name = models.CharField(max_length=100, blank=True)  # Add a default value
+    description = models.CharField(max_length=100, blank=True)
+
+    class Meta:
+        abstract = True
+
+
+class AbstractIPAddress(models.Model):
+    ip_address = models.GenericIPAddressField()
+    description = models.CharField(max_length=100, blank=True)
+    # subnet = models.ForeignKey(swapper.get_model_name("django_ipam", "Subnet"))
```

Comment:
why it is commented? You can't get it working?

Reply:
Yeah, I am still trying to figure it out, as of now I felt it can be solved in the next PR, anyways will try to fix it in this one itself.

Reply:
Do I make use of a GenericForeginKey over here ?

Reply:
no, let's keep it simple now